napari/layers/vectors/vectors.py

```diff
@@ -125,31 +127,20 @@ class Vectors(Layer):
     out_of_slice_display : bool
         If True, renders vectors not just in central plane but also slightly out of slice
         according to specified point marker size.
+    fixed_canvas_width : bool
+        If True, vector width is defined in canvas pixels.
```

Comment:
In data space. In practice:
- `True`: vectors maintain their screen-space thickness no matter the zoom (literally pixel width). This is useful when the width is just a visualization aid (In most applications it should be that, I think) rather than having a physical meaning.
- `False`: vector width has spatial meaning (i.e: it's a "stick"). This is now in data space: for example, a vector of thickness 10 should be as thick as 10 image voxels, assuming no transforms.